client/js/verification-portal/view.js

```diff
@@ -68,11 +57,15 @@ const styles = {
 		position: 'relative',
 	},
 	leftPositionButton: {
-		width: '100%',
+		'padding-top': '50px',
 		position: 'absolute',
 		left: '1%',
 		'z-index': '100',
 	},
+	rightPositionButton: {
+		float: 'right',
+		'padding-right': '50px',
+	},
```

Comment:
The Out of Business button was in the middle of the form, covering up the General Info title. I moved it back to the right side.